src/js/stream.js

```diff
@@ -20,12 +21,20 @@ class Stream {
 	init () {
 		return Promise.all([this.renderComments(), this.getJsonWebToken()])
 			.then(() => {
+				if (!this.token && this.userIsSignedIn) {
+					this.renderDisplayNameOverlay();
+				}
+
 				if (this.token && this.embed) {
 					this.login();
 				}
 			});
 	}
 
+	static renderDisplayNameOverlay () {
```

Comment:
This was done to make it easier to test that `displayNameOverlay()` is called and this way we don't have to rely on dispatching and listening for an event. I'm totally open to other solutions to this problem!

Reply:
Also, because this function doesn't use `this`, according to Origami specs, it should be a static function.

Reply:
Able to successfully stub `displayNameOverlay` so this function isn't needed any more!